great_expectations/data_context/types/base.py

```diff
@@ -540,6 +540,10 @@ class Meta:
     skip_inapplicable_tables = fields.Boolean(required=False, allow_none=True)
     batch_spec_passthrough = fields.Dict(required=False, allow_none=True)
 
+    # Inferred Glue Catalog
```

Comment:
@lccasagrande Got it -- these fields make sense.  Thank you.

Reply:
Fixed